File: java/src/main/java/com/gaston/macbook/simplemvp/data/net/ApodApiService.java
package com.gaston.macbook.simplemvp.data.net;

import com.gaston.macbook.simplemvp.presentation.show_apod.model.Apod;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface ApodApiService {

    @GET("apod")
    Call<Apod> getApodData(@Query("api_key") String API_KEY);
}

File: kotlin/src/main/java/com/gaston/macbook/kotlinsimplemvp/data/net/ApodApiService.kt
package com.gaston.macbook.kotlinsimplemvp.data.net

import com.gaston.macbook.kotlinsimplemvp.presentation.show_apod.model.Apod
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface ApodApiService {

    @GET("apod")
    fun getApodData(@Query("api_key") API_KEY: String): Call<Apod>
}